diff --git a/src/process/index.js b/src/process/index.js
index 97ea651..937a5d4 100644
--- a/src/process/index.js
+++ b/src/process/index.js
@@ -60,7 +60,7 @@ export default class ProcessServer {
 
           ids.push(id);
           if (!timestamps[id]) {
-            log('detected game!', name);
+            log('detected game!', name, id);
             timestamps[id] = Date.now();
           }
 
diff --git a/src/transports/ipc.js b/src/transports/ipc.js
index 91d1eb2..550da10 100644
--- a/src/transports/ipc.js
+++ b/src/transports/ipc.js
@@ -219,6 +219,8 @@ export default class IPCServer {
 
         socket.close(ErrorCodes.INVALID_CLIENTID);
         return;
+      } else {
+        log('connection', clientId)
       }
 
       socket.on('error', e => {
